.claude/requirements/done/REQ-005.md

```
@@ -0,0 +1,23 @@
+# [REQ-005] League History
+
+## 개요
+유저의 시즌별 최종 티어 기록을 프로필 내 리그 탭에서 확인할 수 있도록 노출한다.
+
+## API
+| Method | Path | 설명 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)